src/libraries/Microsoft.PowerFx.LanguageServerProtocol/Protocol/CodeActionCommand.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.PowerFx.LanguageServerProtocol.Protocol
+{
+    /// <summary>
+    /// Code action command object model
+    /// </summary>
+    public class CodeActionCommand
+    {
+        /// <summary>
+        /// ctor
+        /// </summary>
+        public CodeActionCommand()
+        {
+            Arguments = new object[] { };
+        }
+
+        /// <summary>
+        /// Gets or sets title of the command.
+        /// </summary>
+        public string Title { get; set; }
+
+        /// <summary>
+        /// Gets or sets command to be executed.
+        /// </summary>
+        public string Command { get; set; }
+
+        /// <summary>
+        /// Gets or sets command arguments.
+        /// </summary>
+        public object[] Arguments { get; set; }
```

Comment:
>object [](http://example.com/codeflow?start=15&length=6)

can this be a stronger type than object?

Reply:
Removed as no need to expose.

Reply:
I see the comment, but no code changes. Are the latest changes pushed?

Reply:
Referred to CodeAction class.

Reply:
See [https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.languageserver.protocol.command.arguments?view=visualstudiosdk-2019#Microsoft_VisualStudio_LanguageServer_Protocol_Command_Arguments](https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.languageserver.protocol.command.arguments?view=visualstudiosdk-2019#Microsoft_VisualStudio_LanguageServer_Protocol_Command_Arguments)